plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -357,8 +386,50 @@ module.exports = class ServiceTree extends Tree {
     if (status == null) {
       return null;
     }
+    const statusIconCounts = this.getStatusIconCounts();
+    if (statusIconCounts.total <= 1) {
+      return status.displayName;
+    }
+    const statusSummary = {
+      id: "",
+      status: status.displayName,
+      statusCounts: statusIconCounts.status,
+      values: {
+        priorityStatusCount: statusIconCounts.status[status.icon.name],
+        totalCount: statusIconCounts.total
+      }
+    };
+    switch (status.icon) {
+      case StatusIcon.SUCCESS:
+        statusSummary.id = i18nMark(
+          "Running ({priorityStatusCount} of {totalCount})"
```

Comment:
It's always `"Something ({priorityStatusCount} of {totalCount})"`, can we add `({priorityStatusCount} of {totalCount})` after the switch?

Reply:
This would also reduce the number of similar messages in the message catalog.

Reply:
Maybe? I was unsure which was better for translations, the whole statement or the status and `({priorityStatusCount} of {totalCount})` separately.